### DISCUSSION TITLE
Newest Debian build returning errors while installing (version 3.8.0-1)

### DISCUSSION THREAD

**Discussion by Priff13** [logisim-evolution/logisim-evolution#1600] [Q&A] [answered]
I am having trouble installing the app from the Debian compiled form, but I kept getting errors whilst installing.
When I tried to install the app from the .deb file through my folder system, it returned the error:
```
The following packages have unmet dependencies:
  logisim-evolution: Depends: libc6 but it is not installable
                     Depends: zlib1g but it is not installable
```
And when I tried to install it via terminal with the command:
```
sudo dpkg -i logisim-evolution_3.8.0-1_amd64.deb
```
it returned the error:
```
dpkg-deb: error: archive 'logisim-evolution_3.8.0-1_amd64.deb' uses unknown compression for member 'control.tar.zst', giving up
dpkg: error processing archive logisim-evolution_3.8.0-1_amd64.deb (--install):
 dpkg-deb --control subprocess returned error exit status 2
Errors were encountered while processing:
 logisim-evolution_3.8.0-1_amd64.deb
```
I have reviewed and re-installed both libc6 and zlib1g, but it still returns those errors.
Keep in mind that I am running this on the Chrome OS Linux developer environment (uses Debian).
Any suggestions on how to fix these errors would be highly appreciated.

**Comment by maehne** [comment, marked as answer]
The DEB package provided as Download is currently built on Ubuntu and therefore unfortunately doesn't work on Debian. To resolve the issue, I suggest that you install `openjdk-17-jdk` using `apt` and build the package from source as explained in the README.

**Reply by maehne** [reply]
FWIW, the issue is tracked in #1569.
